ci_tools/check_slots.py

```diff
@@ -101,11 +100,11 @@ def fill_dictionary(title, message, file, start_line, end_line, annotation_level
                         line_num += 1
                         line = lines[line_num]
                         end_line = line_num + 1
-                    ErrorCollection['non_alphabetical_all'].append(fill_dictionary("Non-alphabetical `__all__`", f"`pyccel.ast.{mod_name}`",
+                    error_collection['non_alphabetical_all'].append(fill_dictionary("Non-alphabetical `__all__`", f"pyccel.ast.{mod_name}",
                         inspect.getfile(mod), start_line, end_line, "warning", f"Sort the __all__ attribute of `{mod_name}`"))
                     break
     else:
-        ErrorCollection['missing_all'].append(fill_dictionary("Missing `__all__`", f"`pyccel.ast.{mod_name}`",
+        error_collection['missing_all'].append(fill_dictionary("Missing `__all__`", f"pyccel.ast.{mod_name}",
```

Comment:
Can we keep the quotes around the code (you might want to add them inside `fill_dictionary` to make your life easier